go/pkg/rexec/rexec.go

```diff
@@ -384,6 +385,30 @@ func (ec *Context) DownloadOutputs(outputDir string) {
 	}
 }
 
+// GetOutputFileDigests returns a map of output file paths to digests.
+// This function is supposed to be run after a successful cache-hit / remote-execution
+// has been run with the given execution context. If called before the completion of
+// remote-execution, the function returns a nil result.
+func (ec *Context) GetOutputFileDigests(useAbsPath bool) (map[string]digest.Digest, error) {
```

Comment:
useAbsPath is convenient but does it make sense? You can arguably parse the output with other tools.

I'm not exactly opposed to it but I think long term it'd work better as a SDK wide option (eg show full path on outputs for convenience).

Reply:
I mean currently I don't have a usecase for the absolute paths, I only want relative paths. That said, I think it is a useful piece of functionality for this method to optionally return either the relative / absolute paths for the output files. There are going to be multiple users for the SDK.

Reply:
Right, I am just wondering if this is the correct approach vs a SDK Client option for all places we show output back to the user :)

... but as usual, we can decide later.